Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -814,7 +814,8 @@ namespace Babylon
                     env,
                     JS_CLASS_NAME,
                     {
-                        InstanceAccessor("pointerId", &PointerEvent::GetPointerId, nullptr)
+                        InstanceAccessor("pointerId", &PointerEvent::GetPointerId, nullptr),
+                        InstanceAccessor("pointerType", &PointerEvent::GetPointerType, nullptr)
```

Comment:
Given that this object will receive its values on instantiation, the values won't change, and the values are primarily read on the JS side, you could do something like this:

```c++
// This part is optional. 
// If you do decide to have these in the class definition, 
// the napi_writable is necessary to allow assignment in the constructor.
{
    InstanceValue("pointerId", env.Undefined(), napi_writable),
    InstanceValue("pointerType", env.Undefined(), napi_writable),
}

...

PointerEvent(const Napi::CallbackInfo& info)
    : Napi::ObjectWrap<PointerEvent>{info}
{
    auto thisObject = info.This().As<Napi::Object>();
    const auto params = info[1].As<Napi::Object>();
    
    thisObject.Set("pointerId", params.Get("pointerId"));
    m_pointerId = params.Get("pointerId").As<Napi::Number>().Int32Value();

    thisObject.Set("pointerType", params.Get("pointerType"));
    m_pointerType = params.Get("pointerType").As<Napi::String>().Utf8Value();
}
```

And then no more need for the accessor functions, and no more communication over the wire for static values.